launcher/Application.h

```diff
@@ -74,6 +76,13 @@ class ITheme;
 class MCEditTool;
 class ThemeManager;
 class IconTheme;
+class BaseInstance;
+
+struct MinecraftTarget;
+
+// pointers for lazy people
+using InstancePtr = std::shared_ptr<BaseInstance>;
+using MinecraftTargetPtr = std::shared_ptr<MinecraftTarget>;
```

Comment:
I will instead nitpick on this latter definition: it's declaring *another* way to write MinecraftTarget::Ptr... I'd prefer to fix the other declaration to be the same